apps/examples/wifi_test/wifi_test.c

```diff
@@ -248,7 +249,9 @@ int main(int argc, FAR char *argv[])
 int wifi_test_main(int argc, char *argv[])
 #endif
 {
-
+#if CONFIG_TASH
+	tash_cmd_install("wifi_scan", wifi_test_proc, TASH_EXECMD_SYNC);
```

Comment:
This context is different from the usual method of specifying an entrypoint via TASH. Here, wifi_test_main is an entrypoint which is added as a builtin TASH command. However, we would like to invoke wifi_test_proc 'only' from a TASH console. Hence the use of the tash install function within this entrypoint.